docker/rootfs/root/_setup-storage.sh

```diff
@@ -0,0 +1,20 @@
+#!/command/with-contenv bash
+
+uid=$(s6-envuidgid -i -u nobody importas UID UID s6-echo '$UID')
+gid=$(s6-envuidgid -i -g nobody importas GID GID s6-echo '$GID')
+legacy=("webtlo.db" "config.ini" "logs")
```

Comment:
I also have `tfiles` directory, should it be moved too?

Reply:
Yes, I suppose.